### PR TITLE
Add xwidget

### PR DESCRIPTION
**Pull request by rynffoll**

### CODE REVIEW

File: modes/xwidget/evil-collection-xwidget.el [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+;;;###autoload
+(defun evil-collection-xwidget-setup ()
+  "Set up `evil' bindings for `xwidget'."
+  (evil-collection-define-key 'normal 'xwidget-webkit-mode-map
+    "q" 'quit-window
+    "k" 'xwidget-webkit-scroll-down-line
+    "j" 'xwidget-webkit-scroll-up-line
+    "h" 'xwidget-webkit-scroll-backward
+    "l" 'xwidget-webkit-scroll-forward
+    (kbd "C-f") 'xwidget-webkit-scroll-up
```
Comment (jojojames) [inline modes/xwidget/evil-collection-xwidget.el]: C-u and C-d?

I'd gate the C-u with the evil-want-C-u variable (don't remember if there's one for C-d too).

File: modes/xwidget/evil-collection-xwidget.el [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    (kbd "C-b") 'xwidget-webkit-scroll-down
+    "+" 'xwidget-webkit-zoom-in
+    "=" 'xwidget-webkit-zoom-in
+    "-" 'xwidget-webkit-zoom-out
+    "R" 'xwidget-webkit-reload
+    "gr" 'xwidget-webkit-reload
+    "H" 'xwidget-webkit-back
```
Comment (jojojames) [inline modes/xwidget/evil-collection-xwidget.el]: Maybe > and <?

File: modes/xwidget/evil-collection-xwidget.el [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    "L" 'xwidget-webkit-forward
+    "gu" 'xwidget-webkit-browse-url
```
Comment (jojojames) [inline modes/xwidget/evil-collection-xwidget.el]: gu seems okay but if you take off "L", maybe it'd have a place there. Up to you though.

**Comment by jojojames** [comment]
Do you have a link to this webkit package? This seems interesting!

And thanks for the PR!

File: modes/xwidget/evil-collection-xwidget.el (hunk 2 above)
Comment (rynffoll) [reply modes/xwidget/evil-collection-xwidget.el]: I was inspired by the hotkeys of Vimium, which uses `H/L` to navigate through history.

**Comment by rynffoll** [comment]
> Do you have a link to this webkit package? This seems interesting!
> 
> And thanks for the PR!

It's built-it package (`--with-xwidgets`).
https://www.gnu.org/software/emacs/manual/html_node/elisp/Xwidgets.html

**Comment by jojojames** [comment]
Thanks!

**Pull request merged**